package.json

```diff
@@ -19,7 +19,7 @@
   ],
   "dependencies": {
     "aes-js": "^3.1.2",
-    "bfx-api-node-core": "^1.6.0",
+    "bfx-api-node-core": "git+https://github.com/avsek477/bfx-api-node-core.git#auto-resubscribe-fix",
```

Comment:
needs to be changed after merged: https://github.com/bitfinexcom/bfx-api-node-core/pull/34